gax/src/test/java/com/google/api/gax/rpc/SpoolingCallableTest.java

```diff
@@ -69,7 +69,7 @@ public void testHappyPath() throws InterruptedException, ExecutionException {
     call.getController().getObserver().onResponse("response2");
     call.getController().getObserver().onComplete();
 
-    assertThat(result.get()).containsAllOf("response1", "response2").inOrder();
+    assertThat(result.get()).containsAtLeast("response1", "response2").inOrder();
```

Comment:
Should this be a better option: `assertThat(result.get()).containsExactly("response1", "response2").inOrder();` ?

Reply:
Done